### PR TITLE
fix: dedicated deployment for mktool customer

### PR DESCRIPTION
**Pull request by aashishmalik**
## What are the changes introduced in this PR?

1. Merges external_ids, emails, and phones for entries with the same subscription_group_id and subscription_state
2. batch size set to 25

## What is the related Linear task?

Resolves INT-XXX

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3901/test-report.html

**Review by utsabc — DISMISSED** [review]

File: test/integrations/destinations/braze/router/data.ts [hunk 1]
```diff
@@ -299,13 +327,13 @@ export const data = [
                     JSON: {
                       subscription_groups: [
                         {
-                          external_id: ['user123'],
+                          external_ids: ['user123', 'user12345'],
```
Comment (utsabc) [inline test/integrations/destinations/braze/router/data.ts:330]: I see in API its `external_id` not `external_ids`
ref: https://www.braze.com/docs/api/endpoints/subscription_groups/post_update_user_subscription_group_status/#request-parameters

Comment (aashishmalik) [reply test/integrations/destinations/braze/router/data.ts:330]: using V2

**Review by utsabc — APPROVED** [review]

**Review by yashasvibajpai — APPROVED** [review]

File: test/integrations/destinations/braze/router/data.ts (hunk 1 above)
Comment (aashishmalik) [reply test/integrations/destinations/braze/router/data.ts:330]: using V2 now

**Pull request merged**